sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/npm.yaml

```
@@ -0,0 +1,45 @@
+provider:
+  metadata:
+    name: NPM
+    category: Package repository
+    message: Make sure this NPM API token gets revoked, changed, and removed from the code.
```

Maybe you'll want to change this to match the rule title.
```suggestion
    message: Make sure this NPM access token gets revoked, changed, and removed from the code.
```
or the other way around.